modules/@apostrophecms/piece-type/index.js

```diff
@@ -1060,6 +1060,94 @@ module.exports = {
             });
           }
         }
+      },
+
+      localize: {
+        usage: 'Add draft version documents for each locale when a module has the "localized" option.' +
+        '\nExample: node app [moduleName]:localize',
+        async task() {
+          if (!self.options.localized) {
+            throw new Error('Localized option not set to true, so the module cannot be localized.');
+          }
+
+          console.log('Adding drafts and locales to documents');
+
+          const locales = Object.keys(self.apos.i18n.locales);
+          const lastPublishedAt = new Date();
+          const inserts = [];
+          const deletes = [];
+
+          await self.apos.migration.eachDoc({ type: self.name }, async doc => {
+            if (doc.aposDocId && !doc._id.endsWith('published') && !doc._id.endsWith('draft')) {
+              deletes.push(doc._id);
+
+              for (const locale of locales) {
+                const newDraft = {
+                  ...doc,
+                  aposLocale: `${locale}:draft`,
+                  aposMode: 'draft',
+                  aposDocId: doc._id,
+                  _id: `${doc.aposDocId}:${locale}:draft`
+                };
+                const newPublished = {
+                  ...doc,
+                  aposLocale: `${locale}:published`,
+                  aposMode: 'published',
+                  aposDocId: doc._id,
+                  _id: `${doc.aposDocId}:${locale}:published`,
+                  lastPublishedAt
+                };
+                inserts.push(newDraft);
+                inserts.push(newPublished);
+              }
+            }
+          });
+
+          await self.apos.doc.db.deleteMany({ _id: { $in: deletes }});
```

Comment:
This is a little *too* efficient (: You could run out of RAM and fail on a large database. `insertMany` is good, but only once per document, after the loop over locales.